engine/src/main/resources/templates/ResourceOptimizationService/insightsEmailBody.html

```diff
@@ -0,0 +1,547 @@
+      }
+
+      .rh-card.rh-m-set-height {
+        height: auto;
+      }
+    }
+
+  </style>
+|}
+</head>
+
+<body leftmargin="0" marginwidth="0" topmargin="0" marginheight="0" offset="0" class="rh-body">
+<center>
+    <table align="center" border="0" cellpadding="0" cellspacing="0" height="100%" width="100%" class="rh-body-table">
+        <tr>
+            <td align="center" valign="top" class="rh-body-cell">
+                <table border="0" cellpadding="0" cellspacing="0" class="rh-template-container">
+
+                    <!-- Head -->
+                    <tr>
+                        <td class="rh-head">
+                            <table class="rh-masthead" role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+                                <tr>
+                                    <!--Red Hat logo-->
+                                    <td align="center" class="rh-masthead__brand" bgcolor="#151515">
+                                        <a href="https://console.redhat.com/" target="_blank" class="rh-masthead__brand-link">
+                                            <img src="https://console.redhat.com/apps/frontend-assets/email-assets/logo_insights.jpg" alt="Red Hat logo" width="182" height="90" class="rh-masthead__brand-img" />
+                                        </a>
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td class="rh-masthead__subhead">
+                                        <h1><b>{#insert content-title}{/}</b></h1>
+                                    </td>
+                                </tr>
+                            </table>
+                        </td>
+                    </tr>
+                    <!-- end head -->
+
+                    <!-- Body section -->
+                    <tr>
+                        <td class="rh-body">
+                            <table class="rh-content" role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+                                {#insert content-body}{/}
+                            </table>
+                        </td>
+                    </tr>
+                    <!-- end body section -->
+
+                    <!-- Footer -->
+                    <tr>
+                        <td class="rh-footer">
+                            <table role="presentation" border="0" align="center" cellpadding="0" cellspacing="0" width="100%">
+                                <tr>
+                                    <td class="rh-footer__text">
+                                        You received this notification because of selections you made in your <a href="https://console.redhat.com/user-preferences/notifications/rhel" target="_blank">Hybrid Cloud Console User Preferences</a>. You can opt in or out of receiving notifications by changing your User Preferences.
```

Comment:
```suggestion
                                        You received this notification because of selections you made in your <a href="{environment.url}/user-preferences/notifications/rhel" target="_blank">Hybrid Cloud Console User Preferences</a>. You can opt in or out of receiving notifications by changing your User Preferences.
```